WordPress/Classes/Services/BloggingPromptsService.swift

```diff
@@ -179,17 +179,44 @@ class BloggingPromptsService {
 
     // MARK: - Init
 
+    /// Initializes a service for blogging prompts.
+    ///
+    /// - Parameters:
+    ///   - contextManager: The CoreDataStack instance.
+    ///   - remote: When supplied, the service will use the specified remote service.
+    ///     Otherwise, a remote service with the default account's credentials will be used.
+    ///   - blog: When supplied, the service will perform blogging prompts requests for this specified blog.
+    ///     Otherwise, this falls back to the default account's primary blog.
     required init?(contextManager: CoreDataStackSwift = ContextManager.shared,
                    remote: BloggingPromptsServiceRemote? = nil,
                    blog: Blog? = nil) {
-        guard let account = try? WPAccount.lookupDefaultWordPressComAccount(in: contextManager.mainContext),
-              let siteID = blog?.dotComID ?? account.primaryBlogID else {
+        let blogObjectID =  blog?.objectID
```

Comment:
🧐 There is an extra space here.

Reply:
Good eye! 🦅. Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/20854/commits/a9d5782042c7cc1cc58a2a755c14d2cd1b084093.